features/gezag-minderjarige/bevoegdheid-tot-gezag.feature

```diff
@@ -0,0 +1,112 @@
+# language: nl
+
+Functionaliteit: Gezag bepalen voor een minderjarige wanneer een of beide ouders niet bevoegd zijn tot gezag
+  Een persoon is onbevoegd tot gezag wanneer die:
+  - is overleden
+  - onder curatele staat
+  - is minderjarig
+
+
+  Achtergrond:
+    Gegeven de persoon 'Gerda' met burgerservicenummer '000000012'
+    * is meerderjarig
+    En de persoon 'Aart' met burgerservicenummer '000000024'
+    * is meerderjarig
+    En de persoon 'Ariana' met burgerservicenummer '000000036'
+    * is meerderjarig
+    En de persoon 'Bert' met burgerservicenummer '000000048'
+    * is minderjarig
+
+
+  Regel: Er is sprake van eenhoofdig ouderlijk gezag als één van de ouders met tweehoofdig ouderlijk gezag overleden of niet bevoegd is
+  
+    Scenario: Minderjarige heeft twee ouders en één van de ouders staat onder curatele
+      Gegeven 'Gerda' en 'Aart' zijn getrouwd
+      En persoon 'Bert'
+      * heeft 'Gerda' als ouder
+      * heeft 'Aart' als ouder
+      En 'Gerda' staat onder curatele
+      Als gezag wordt gevraagd met het burgerservicenummer van 'Bert'
```

Comment:
In de Als stap kan `met het burgerservicenummer` kan volgens mij weg. Het is niet nodig voor het begrip van de lezer

Reply:
ook in de andere plekken en features doorgevoerd